website/docs/guides/adapter-creation.md

```diff
@@ -186,20 +186,20 @@ One of the most important choices you will make during the cookiecutter generati
-|---------------------------------------------------|-------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `./setup.py`                                      | `setup()` function                                                | adapter meta-data (package name, version, author, homepage, etc)                                                                                                                      |
-| `myadapter/dbt/adapters/myadapter/__init__.py`    | `AdapterPlugin`                                                   | bundle all the information below into a dbt plugin                                                                                                                                    |
-| `myadapter/dbt/adapters/myadapter/connections.py` | `MyAdapterCredentials` class                                      | parameters to connect to and configure the database, via a the chosen Python driver                                                                                                   |
-| `myadapter/dbt/adapters/myadapter/connections.py` | `MyAdapterConnectionManager` class                                | telling dbt how to interact with the database w.r.t opening/closing connections, executing queries, and fetching data. Effectively a wrapper around the db API or driver.             |
-| `myadapter/dbt/include/bigquery/`                 | a dbt project of macro "overrides" in the format of "myadapter__" | any differences in SQL syntax for regular db operations will be modified here from the global_project (e.g. "Create Table As Select", "Get all relations in the current schema", etc) |
-| `myadapter/dbt/adapters/myadapter/impl.py`        | `MyAdapterConfig`                                                 | database- and relation-level configs and                                                                                                                                              |
-| `myadapter/dbt/adapters/myadapter/impl.py`        | `MyAdapterAdapter`                                                | for changing _how_ dbt performs operations like macros and other needed Python functionality                                                                                          |
-| `myadapter/dbt/adapters/myadapter/column.py`      | `MyAdapterColumn`                                                 | for defining database-specific column such as datatype mappings                                                                                                                       |
+| File | Component     | <div style={{width:'350px'}}>Purpose</div>    |
```

Comment:
Not for this PR but we should open an Asana task to fix the scrolling. The table is wider than the page so you have to scroll to view, but the way it's spaced the visuals just end and the `Next` button is off the screen. 

<img width="845" alt="Screenshot 2024-02-29 at 2 45 41 PM" src="https://github.com/dbt-labs/docs.getdbt.com/assets/60105315/3454b573-d2ca-482a-9de8-31ff8a84012c">
<img width="981" alt="Screenshot 2024-02-29 at 2 45 33 PM" src="https://github.com/dbt-labs/docs.getdbt.com/assets/60105315/18e2985a-72fb-439c-9ec0-15643a5a1cb9">
<img width="1221" alt="Screenshot 2024-02-29 at 2 45 26 PM" src="https://github.com/dbt-labs/docs.getdbt.com/assets/60105315/df1c29ce-5050-4b2d-aeab-1c44d9709990">